worksheet_B/Mandelbrot.cpp

```diff
@@ -3,10 +3,22 @@
 
 #include "stdafx.h"
 
+float ImageWidth = 800.0;
+float ImageHeight = 800.0;
+
+int Iteration = 0;
+int maxIteration = 200;
+
+int Red = 0;
+int Green = 0;
+int Blue = 0;
+int minValue = 0;
+int maxValue = 255;
```

Comment:
Do all of these need to be global variables? Probably better to declare them inside `main()`